thinknimble/tn-spa-bootstrapper
# reorganize services in our react and react native clients

## What this does

Cleans up the react side of the bootstrapper by reorganizing some folders and aligning similarities between the Navite and Web portions 

- adds two new profiles for expo to build (both are dev but one builds for simulator)
- removes unnecessary vars from npm run start (sentry does not work in expo run so unless you are running a development build it wont work)
- reorganizes the services folder
- aligns the axios instance between both apps

The aim is to bring all 3 frameworks as close to each other as possible


## How to test

- create an account (on mobile tn-forms appears to have a bug we need to check out)
- login 
**I need to add some checks to ignore files when using gql still (separate PR)**
